tests/deprecated_api/test_remove_1-8.py

```diff
@@ -962,3 +963,23 @@ def test_trainer_num_gpu_0(monkeypatch, gpus, expected_num_gpus, strategy):
         " Please use `Trainer.num_devices` instead."
     ):
         assert Trainer(gpus=gpus, strategy=strategy).num_gpus == expected_num_gpus
+
+
+@pytest.mark.parametrize(
+    ["trainer_kwargs", "expected_ipus"],
+    [
+        ({}, 0),
+        ({"devices": 1}, 0),
```

Comment:
What's the purpose of these two sets of parameters? If we are testing for IPU Accelerator.

Reply:
To test the branch that when IPU is not used Trainer.ipus should return 0